code/modules/admin/smites/imaginary_friend_special.dm

```diff
@@ -58,7 +58,6 @@
 		return FALSE
 
 	var/list/volunteers = SSpolling.poll_ghost_candidates(
-		question = "Do you want to play as an imaginary friend?",
```

Comment:
Why are we removing the question here?

Reply:
@JohnFulpWillard the "Do you want to play as X" style question gets auto-generated if there's role_name_text and no question